sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -308,6 +322,17 @@
         .build();
   }
 
+  /** Like {@link #readGenericRecords(Schema)}, but for {@link FileIO.ReadableFile} collections. */
```

Comment:
Probably clearer to say: "but for a {@link PCollection} of {@link FileIO.ReadableFile}, for example, returned by {@link FileIO#readMatches}."

Reply:
done